packages/gestalt/src/TextField.js

```diff
@@ -77,9 +77,9 @@ type Props = {|
    */
   tags?: $ReadOnlyArray<Element<typeof Tag>>,
   /**
-   * The type of input. For non-telephone numerical input, please use [NumberField](https://gestalt.pinterest.systems/numberfield).
```

Comment:
This comment should remain. We need to guide users to the NumberField component.